addons/project/views/project_views.xml

```diff
@@ -263,23 +263,23 @@
                                     <label for="legend_normal" string=" " class="o_status mt4"
                                         title="Task in progress. Click to block or set as done."
                                         aria-label="Task in progress. Click to block or set as done." role="img"/>
-                                    <div class="col-11 ps-0">
+                                    <div class="col-11 ps-2 ps-0">
```

Comment:
```suggestion
                                    <div class="col-11 ps-2">
```
? Or am I missign something?

Reply:
Due to the stable version policy, we can't remove the class of ''

Reply:
My bad :+1:

Reply:
I think it is better to remove that class as @auon-odoo suggested to be sure the class you added is used and it would easier the padding we want to apply on that element when we read the code.

Reply:
Done Thanks :+1: